app/controllers/vehicles_controller.rb

```diff
@@ -7,4 +7,13 @@ def show
     @vehicle = Vehicle.find(params[:id])
   end
 
+  def new
+
+  end
+
+  def create
+    Vehicle.create(name:params[:name],passenger_capacity:params[:passenger_capacity],locked:params[:locked])
```

Comment:
I feel like there is a shorter way to do this. I think it might look something like this but I am not 100%. We will probably see it in class next week.

Vehicle.create(vehicle_params)

private
  def vehicle_params
    params.permit(:name, :passenger_capacity, :locked)
  end

Reply:
I'll look more into that.

Reply:
I was able to implement this in my user story 3 PR so it is much cleaner.

`def create
    Port.create(port_params)
    redirect_to '/ports'
  end

  private

  def port_params
    params.require(:port).permit(:name, :dock_count, :panamax)
  end`